maven-plugin/src/main/scala/akka/grpc/maven/AbstractGenerateMojo.scala

```diff
@@ -203,11 +203,13 @@ abstract class AbstractGenerateMojo @Inject() (buildContext: BuildContext) exten
       targets: Seq[Target]): Int =
     try {
       val incPath = "-I" + protoDir.getCanonicalPath
-      protocbridge.ProtocBridge.run(
-        protocCommand,
+      protocbridge.ProtocBridge.execute(
+        ProtocRunner.fromFunction((args, _) => protocCommand(args)),
         targets,
         Seq(incPath) ++ protocOptions ++ schemas.map(_.getCanonicalPath),
-        pluginFrontend = protocbridge.frontend.PluginFrontend.newInstance)
+        artifact =>
+          throw new RuntimeException(
+            s"The version of sbt-protoc you are using is incompatible with '${artifact}' code generator. Please update sbt-protoc to a version >= 0.99.33"))
```

Comment:
We'll do the sbt-protoc upgrade, will this still apply?

Reply:
Yes, recent versions of sbt-protoc should not touch this parameter.

(this logic was inside sbt-protoc before, but that constructor became private, so I applied it here as well)